packages/plugin-dom-editable/src/DomEditable.ts

```diff
@@ -143,11 +143,21 @@ export class DomEditable<T extends JWPluginConfig = JWPluginConfig> extends JWPl
             }
             if (!processed) {
                 for (const action of batch.actions) {
-                    const commandSpec = this._matchCommand(action);
-                    if (commandSpec) {
-                        const [commandName, commandParams] = commandSpec;
-                        if (commandName) {
-                            await execCommand(commandName, commandParams);
+                    if (action.type === '@redraw') {
+                        domEngine.markForRedraw(action.domNodes);
+                        await domEngine.redraw({
+                            add: [],
+                            move: [],
+                            remove: [],
+                            update: [],
+                        });
```

Comment:
Je pense qu'il faut changer redraw pour que ce ne soit pas requis de remplir ces trucs là. C'est un peu out of scope donc si c'est pas ridiculously easy de le faire alors tant pis.